crates/core/src/time.rs

```diff
@@ -362,3 +375,34 @@ impl From<DurationNanos> for crate::tendermint::Timeout {
         Self::from(std::time::Duration::from(val))
     }
 }
+
+#[cfg(test)]
+mod core_time_tests {
+    use super::*;
+
```

Comment:
These tests demonstrate that `DateTimeUtc::from_rfc3339` and `DateTimeUtc::to_rfc3339` are inverses. They also demonstrate that valid `DateTime`s are not necessarily valid `DateTimeUtc`s. Could we please also have a test to demonstrate that valid `DateTimeUtc`s are valid `DateTime`s? This would increase confidence that `DateTimeUtc` is a subset of `DateTime`, and that the hardware wallet will successfully handle it.